memoize/memoized.py

```diff
@@ -93,7 +93,7 @@ def wrappedFunc(*args, **kwargs):
                     print(e)
                     ret = None
                 resultFile.close()
-            if ret != None:
+            if not ret is None:
```

Comment:
```suggest
if ret is not None:
```

"A is not B" is better than "not A is B". Idiomatic Python is more readable in this case.